roles/test_operator/README.md

```diff
@@ -3,13 +3,12 @@
 Execute tests via the [test-operator](https://openstack-k8s-operators.github.io/test-operator/).
 
 ## Parameters
-
 * `cifmw_test_operator_artifacts_basedir`: (String) Directory where we will have all test-operator related files. Default value: `{{ cifmw_basedir }}/tests/test_operator` which defaults to `~/ci-framework-data/tests/test_operator`
 * `cifmw_test_operator_namespace`: (String) Namespace inside which all the resources are created. Default value: `openstack`
 * `cifmw_test_operator_index`: (String) Full name of container image with index that contains the test-operator. Default value: `quay.io/openstack-k8s-operators/test-operator-index:latest`
 * `cifmw_test_operator_timeout`: (Integer) Timeout in seconds for the execution of the tests. Default value: `3600`
 * `cifmw_test_operator_logs_image`: (String) Image that should be used to collect logs from the pods spawned by the test-operator. Default value: `quay.io/quay/busybox`
-* `cifmw_test_operator_concurrency`: (Integer) Tempest concurrency value. Default value: `8`
+* `cifmw_test_operator_tempest_concurrency`: (Integer) Tempest concurrency value. As of now this value can not be specified inside `test_vars`. Default value: `8`
```

Comment:
i guess you are chaging  this to conform to a naming convention.

This is pretty disruptive to upstream and downstream CI, so I would avoid changing it unless it blocks adding a concurrency varible for something else.


8 is also unreasonably high as a default. 3-4 is more appropriate.

Reply:
Hi @SeanMooney as you mentioned the change is to conform to the naming convention but without it the patch won't work in the way it is intended to work in, I created patches for all related project including intermediate patch for D/S to keep the vars with both name first and only after to move to the new patch. I believe we can do something similar in Nova project and other projects if needed as well. Regarding the default value I believe @lpiwowar would be the person to discuss that but having that this patch is not related to concurrency directly do I will not changing the default value in this patch.

Reply:
thats fair.

our host vms for the compute nodes tend not to be capable of that level of concurance outside basic API test so that's why i raised it. we likely have already overridden it in most jobs that were impacted. 

that can be discussed seperatly.